tests/test_unsupported.py

```diff
@@ -113,15 +113,19 @@ def main(): r = Rectangle()
 # NOTE: Inclusion here does not indicate that the case is comparable
 EXPECTED_SUCCESSES = [
     "annassign_List.dart",
+    "annassign_List.v",
     "bool_plus_int.jl",
     "bool_plus_int.rs",
+    "bool_plus_int.v",
     "bool_to_int.jl",
     "bool_to_int.nim",
     "bool_to_int.rs",
+    "bool_to_int.v",
     "complex_dict.jl",
     "complex_dict.kt",
     "complex_dict.nim",
     "complex_dict.rs",
+    "complex_dict.v",  # ???
```

Comment:
just a reminder that there are three(3) items in this file that need to be removed/revised/etc before this can be merged

Reply:
@jayvdb I assume you mean `complex_dict.v`, `dict_keys_explicit.v` and `dict_values.v`?

`complex_dict.v` was a false positive due to invalid transpilation of tuples, fixed now. The others are waiting vlang/v#10791 to be merged.

Reply:
I just merged it.